### PR TITLE
Logic and tests

### PR DESCRIPTION
**Pull request by awendt**

### CODE REVIEW

**Review by jtsaito — APPROVED** [review]

File: stale-projects.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+exports.getStaleProjects = (currentProjects, targetProjects) => {
+  return currentProjects.filter(p => !targetProjects.includes(p.node.project.name));
+};
```
Comment (jansiwy) [inline stale-projects.js:3]:
```suggestion
exports.getStaleProjects = (currentProjects, targetProjects) => (
  currentProjects.filter(p => !targetProjects.includes(p.node.project.name))
);
```

File: stale-projects.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+exports.removeStaleProjects = ({client, projects}) => {
+  return Promise.all(projects.map(staleProject => {
+    return client.rest.projects.deleteCard({ card_id: staleProject.node.databaseId })
+  }))
+};
```
Comment (jansiwy) [inline stale-projects.js:9]:
```suggestion
exports.removeStaleProjects = async ({client, projects}) => (
  Promise.all(projects.map(staleProject => (
    client.rest.projects.deleteCard({ card_id: staleProject.node.databaseId })
  )))
);
```

**Comment by awendt** [comment]
@jansiwy I'll add ESLint in a follow-up

File: stale-projects.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+exports.getStaleProjects = (currentProjects, targetProjects) => (
+  currentProjects.filter((p) => !targetProjects.includes(p.node.project.name))
+);
+
+exports.removeStaleProjects = async ({ client, projects }) => (
+  Promise.all(projects.map((staleProject) => (
+    client.rest.projects.deleteCard({ card_id: staleProject.node.databaseId })
+  )))
+);
```
Comment (jansiwy) [inline stale-projects.js:9]: there is a risk of being throttled by the API when running all the requests in parallel. thus, a classic loop with `await` might be better option for this case, i.e. making this less efficient on purpose:

```suggestion
exports.removeStaleProjects = async ({ client, projects }) => {
  for (const project of projects) {
    await client.rest.projects.deleteCard({ card_id: project.node.databaseId });
  }
};
```

**Review by jansiwy — APPROVED** [review]
apart from that: LGTM!

**Pull request merged**